core/tests/test_contacts_crud.py

```diff
@@ -65,8 +66,8 @@ def test_add_note(setup_db):
         ContactPayload(name="Joe", note="Hello World"),
     )
 
-    assert result.message == "Contact created."
-    assert db["Joe"] == Record(name="Joe", note="Hello World")
+    assert result.message == InfoMessages.CONTACT_CREATED.value
+    assert db["Joe"] == Record(name="Joe", notes="Hello World")
```

Comment:
The final step, currently tests are fail because here should be `note="Hello World"`

Reply:
not `notes`